src/darwin/Framework/CHIP/MTRCommissioneeInfo.h

```diff
@@ -57,6 +58,14 @@ MTR_AVAILABLE(ios(18.4), macos(15.4), watchos(11.4), tvos(18.4))
  */
 @property (nonatomic, copy, readonly, nullable) NSDictionary<MTRAttributePath *, NSDictionary<NSString *, id> *> * attributes MTR_PROVISIONALLY_AVAILABLE;
 
+/**
+ * Network interfaces the commissionee has.  If there are no interfaces of a
+ * given type, the corresponding array will be empty.
+ */
+@property (nonatomic, copy, readonly) NSArray<MTRNetworkInterfaceInfo *> * wifiInterfaces MTR_PROVISIONALLY_AVAILABLE;
```

Comment:
I don't think we should cater to weird things like multiple interfaces of the same type in this API? I would just expose the lowest-endpoint-number interface for each type. Another weird (non-compliant) possibility would be an NC cluster that has e.g. WiFi AND Thread feature bits set... What do you think about exposing a `type` property on MTRNetworkInterfaceInfo that will be exactly one of `MTRNetworkCommissioningFeature{WiFi,Thread,Ethernet}NetworkInterface` ? If that subset of the feature mask doesn't equal exactly one of those types then I would ignore the endpoint for purposes of exposing here.